addons/binding/org.openhab.binding.smaenergymeter/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please also add this project to the parent pom (binding/pom.xml), so that the project is included in the build?